### ISSUE TITLE
Question - sub-folders in 'resources' or alternative approach to tracking provenance

### ISSUE THREAD

**Issue by rkboyce** [HL7/PDDI-CDS#1]
￼￼Question: Hi, we are working on value sets for the PDDI CDS IG and wanted to store the RxMix workflows used to create valuesets along with the valueset XMLs. To organize that, we thought of creating a subfolder in resources for each value set group requiring a workflow. It it possible to do that without messing up the functionality having the IG automatically built to include the valuesets?

**Comment by brynrhodes** [comment]
I believe the answer is yes, but it may require some tweaking of how the publisher finds and pulls in the resource content. Would we want to potentially include the RxMix workflows in the IG as supporting documentation as well?

**Comment by rkboyce** [comment]
Yes, we do want to include RxMix workflows in the IG. Based on our discussion last week, it seems that we would want to build an extension to maybe the FHIR valueset 'description' property of valueset resources (please suggest another if more appropriate ) to hold the workflow. I noted this in our minutes. So, perhaps we can close this ticket and create  a new issue to create the extension?
